petclinic-frontend/src/features/carts/components/UserCart.tsx

```diff
@@ -266,57 +269,94 @@ const UserCart = (): JSX.Element => {
   return (
     <div className="UserCart">
       <NavBar />
```

Comment:
In case you're wondering why the Navbar in this page looks different from the other pages, it's because it's applying a css class to the Navbar. You could use an extra div and move the Navbar out or you could also create another Page where you import the Navbar component and the UserCart component. I think the second option is the best since everyone is doing that.